src/scene/materials/standard-material.js

```diff
@@ -109,6 +109,13 @@ let _params = new Set();
  * are specularTint are set, they'll be multiplied by vertex colors.
  * @property {string} specularVertexColorChannel Vertex color channels to use for specular. Can be
  * "r", "g", "b", "a", "rgb" or any swizzled combination.
+ * @property {boolean} useSpecularityFactor Use the specularity factor to weight specularity
+ * @property {number} specularityFactor The factor of specularity, exclusive to specular
+ * @property {Texture|null} specularityFactorMap The factor of specularity as a texture
+ * @property {boolean} f0Tint Multiply F0 reflection color.
```

Comment:
This is part of the public API, and maybe we need to consider more meaningful names, or at least explanations. Are these properties commonly exposed in other engines?

Reply:
But yes, I named it specularityFactor as to not confuse it with the specular flow, as this is actually applied when using metalness.